cloud/main.js

```diff
@@ -237,7 +237,11 @@ Parse.Cloud.define("joinQueue", function(request, response) {
                   }
                 }
               ).then( function(){
-                response.success(player);
+                sendDataChangedNotification().then(function() {
+                  response.success(player);
+                }).fail(function(err) {
+                  response.error(err);
+                });
```

Comment:
I agree with @udiWertheimer if sendNotification fails it shouldn't affect anything